### ISSUE TITLE
Macros do not work

### ISSUE THREAD

**Issue by lonpalmer** [vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT#2]
### What happened?

Using the  Advanced Fantasy Macros for encounters, an error is thrown in the console and no result is returned. 

### What is the expected behaviour?

Macros should produce  encounters in the chat. 

### Relevant Errors and Warnings

```shell
undefined. Cannot read properties of undefined (reading 'index')
[Detected 1 package: ose-advancedfantasytome]
    at OSE.util.wildEncResult (encounterTables.js:145:26)
    at OSE.util.wildEncounter (encounterTables.js:107:16)
    at Macro.eval (eval at #executeScript (foundry.js:22452:16), <anonymous>:3:10)
    at #executeScript (foundry.js:22456:17)
    at Macro.execute (foundry.js:22404:35)
    at MacroConfig._onExecute (foundry.js:74437:17)
OSE.util.wildEncResult @ encounterTables.js:145
OSE.util.wildEncounter @ encounterTables.js:107
eval @ VM977:3
#executeScript @ foundry.js:22456
execute @ foundry.js:22404
_onExecute @ foundry.js:74437
await in _onExecute (async)
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
```

### Additional Support Details

Foundry Version 11
Windows 11
Edge Browser (latest)

### Foundry VTT Core Version

V11

### Old-School Essentials Core Version

1.9.1

### Please update your browser regularly. Which (updated) browsers are you seeing the problem on?

Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by anthonyronda** [comment]
Thanks for the report Lon!

I've identified the issue: a last minute change has made the old name used in the macro incorrect. This is now fixed and awaiting release of version 1.0.1, which is expected in a few hours. I'll close this issue when that release is out

**Comment by anthonyronda** [comment]
Resolved as of 1.0.1
